sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -152,17 +143,15 @@ def online_read(
         requested_features: Optional[List[str]] = None,
     ) -> List[Tuple[Optional[datetime], Optional[Dict[str, ValueProto]]]]:
         online_config = config.online_store
-        assert isinstance(online_config, DynamoDbOnlineStoreConfig)
+        assert isinstance(online_config, DynamoDBOnlineStoreConfig)
         dynamodb = self._initialize_dynamodb(online_config)
 
         result: List[Tuple[Optional[datetime], Optional[Dict[str, ValueProto]]]] = []
         for entity_key in entity_keys:
-            table_instace = dynamodb.Table(table.name)
-            document_id = compute_datastore_entity_id(entity_key)  # TODO check id
-            response = table_instace.get_item(
-                Key={"Row": document_id, "Project": config.project}
-            )
-            value = response["Item"]
+            table_instace = dynamodb.Table(f"{config.project}.{table.name}")
+            document_id = compute_entity_id(entity_key)  # TODO check id
+            response = table_instace.get_item(Key={"Row": document_id})
+            value = response.get("Item", None)
```

Comment:
nit: `None` isn't necessary. The default value is `None` anyways.

Reply:
done